README.md

```diff
@@ -65,6 +65,26 @@ separate character. `shortEmoji` feature allows you to use any emoji in your mes
 converting emoji names to actual Unicode emoji symbols, which are all 1 character long in
 Slack.
 
+#### - `defaultSkinTone` (default: `undefined`)
+When given a number (typically from 2 to 6), adds a skin tone to emoji that support skin tone
+variations. You can always specify a skin tone yourself (`:no_good::skin-tone-6:`), this will
+override the value specified in `defaultSkinTone`. It's best to configure this option in a [local
+config file](#local-config-file).
+
+**CLI usage**:
+```sh
+krasivo hello :no_good: ' ' --default-skin-tone=2
```

Comment:
Why default? Maybe just `--skin-tone`?

Reply:
IMO looks more clear in `~/.krasivorc`. Also, it's overridable (`:no_good::skin-tone-6:`)